pkg/river/internal/value/value.go

```diff
@@ -49,8 +49,11 @@ var Null = Value{}
 // Uint returns a Value from a uint64.
 func Uint(u uint64) Value { return Value{rv: reflect.ValueOf(u), ty: TypeNumber} }
 
-// Int returns a Value from an int64.
-func Int(i int64) Value { return Value{rv: reflect.ValueOf(i), ty: TypeNumber} }
+// Int returns a Value from an int.
+func Int(i int) Value { return Value{rv: reflect.ValueOf(i), ty: TypeNumber} }
+
+// Int64 returns a Value from an int64.
+func Int64(i int64) Value { return Value{rv: reflect.ValueOf(i), ty: TypeNumber} }
```

Comment:
Ditto for removing this; the River representation of numbers will always be either an int64/uint64/float64, even though the decoding of those numbers may downcast when possible.